src/main.cpp

```diff
@@ -529,6 +529,7 @@ void setup()
     digitalWrite(LED_PIN, 1 ^ LED_INVERTED); // turn on for now
 #endif
 
+    
```

Comment:
Please don't check in whitespace changes. i recommend using `trunk fmt` to reformat your changed files.